python/README.md

```diff
@@ -0,0 +1,30 @@
+# Metabase Ruby static embedding sample
+
+This sample code demonstrates using Metabase static embedding in Python using Flask.
+
+## Pre-requisites
+* Have a running instance of Metabase. If you don't have one
+  * [download the free open source version](https://www.metabase.com/start/oss/) or
+  * [sign up for a free trial of Metabase Cloud](https://www.metabase.com/pricing/).
+* A dashboard to embed. If you don't have one, use X-Rays to let Metabase create one for you. Note down the dashboard id.
+
+## Preparing the embed
+1. Sign in to your Metabase instance as an admin. 
+2. Go to admin settings and enabling embedding.
+3. Under admin settings/embedding, click on static embedding and copy the embedding secret key.
+
+## Configure the app
+
+1. Paste the secret key into an env var: `$ export METABASE_EMBEDDING_SECRET="PASTE_SECRET_HERE"`
+2. Create an env var pointing to your Metabase site URL, if it's not on http://localhost:3000 `$ export METABASE_SITE_URL="http://localhost:4000"`
+3. Create an env var with the ID of the dashboard to embed, if it's not 1: `$ export METABASE_EMBED_DASHBOARD_ID="8"`
+
+## Embed the dashboard
+1. Go to your dashboard, click on the share/embed button at the top
+2. Click on the "Publish" button
+
+# Running the sample
+1. Activate the virtual environment: `$ source .venv/bin/activate`
+2. Install the dependencies: `$ pip install -r requirements.txt`
+3. Start the app: `$ flask run -p 9090`
+4. Open the app at http://localhost:9090
```

Comment:
Not related to file, but you will want to remove the python/__pycache__/app.cpython-312.pyc below as that's a binary compilation of this file.

Reply:
I can probably add `__pycache__` to `.gitignore`. Correct?